// to recompile this proto run
// python -m test.protoc -I../protos --python_out=. --grpc_python_out=. ../protos/test.proto

// frontend
// protoc test.proto --js_out=import_style=commonjs:../frontend --grpc-web_out=import_style=commonjs,mode=grpcwebtext:../frontend

syntax = "proto3";

package stream;

service TestService {
    //an example of getting a feature
    rpc TestCall (TestRequest) returns (stream TestResponse) {}
}

// The request containing the user's name
message TestRequest {
  string name = 1;
}

// The response containing the message
message TestResponse {
  string message = 1;
}